FeedService/Services/PiletService.cs

```diff
@@ -0,0 +1,130 @@
+using FeedService.Infrastructure;
+using FeedService.Models;
+using ICSharpCode.SharpZipLib.GZip;
+using ICSharpCode.SharpZipLib.Tar;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace FeedService.Services
+{
+  public class PiletService : IPiletService
+  {
+    private readonly IStorageAccountRepository _storageAccountRepository;
+    public PiletService(IStorageAccountRepository storageAccountRepository)
+    {
+      _storageAccountRepository = storageAccountRepository;
+
+    }
+
+    public async Task<PiletApiResponse> GetLatestPiletsMetaData()
+    {
+      var latestPiletsInfo = await _storageAccountRepository.GetLatestPiletsPackageFiles();
+      var piletResponse = new PiletApiResponse();
+
+      foreach (var pilet in latestPiletsInfo)
+      {
+        var version = ExtractVersion(pilet);
+        var name = ExtractName(pilet);
+        var url = "http://" + Environment.GetEnvironmentVariable("WEBSITE_HOSTNAME") + "/api/pilet/" + name + "/" + version + "/index.js";
```

Comment:
Hm, I think we can also get the right scheme, right? Because usually Azure Functions should be `https` (except using the local emulator).